### PR TITLE
V0.2.0

### PR DESCRIPTION
**Pull request by dnjohnstone**

### CODE REVIEW

File: 03 - Clustering Misorientations.ipynb [hunk 1]
```diff
@@ -165,120 +246,296 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# From Laine et al\n",
+    "sigma7a = Rotation.from_neo_euler(AxAngle.from_axes_angles((1, 0, 0), np.radians(64.40)))\n",
+    "sigma11a = Rotation.from_neo_euler(AxAngle.from_axes_angles((1, 0, 0), np.radians(34.96)))\n",
+    "sigma11b = Rotation.from_neo_euler(AxAngle.from_axes_angles((1, 1, 0), np.radians(85.03)))\n",
```
Comment (pc494) [inline 03 - Clustering Misorientations.ipynb]: I'm a bit confused, why is this `sigma11b` off a different axis `(1,1,0)` to the `sigma11b` from Krakow `(2,1,0)`?

File: 03 - Clustering Misorientations.ipynb [hunk 2]
```diff
@@ -165,120 +246,296 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# From Bonnet et al\n",
+    "sigma7a = Rotation.from_neo_euler(AxAngle.from_axes_angles((2, 1, 0), np.radians(64.62)))\n",
+    "sigma11a = Rotation.from_neo_euler(AxAngle.from_axes_angles((2, 1, 0), np.radians(35.10)))\n",
+    "sigma11b = Rotation.from_neo_euler(AxAngle.from_axes_angles((1, 0, 0), np.radians(84.78)))\n",
+    "sigma13a = Rotation.from_neo_euler(AxAngle.from_axes_angles((2, 1, 0), np.radians(76.66)))\n",
```
Comment (pc494) [inline 03 - Clustering Misorientations.ipynb]: And this `13a` is different too.

File: 03 - Clustering Misorientations.ipynb [hunk 3]
```diff
@@ -165,120 +246,296 @@
...
+    "Calculate difference, defined as minimum rotation angle, between measured and theoretical values"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "difference = Misorientation((~cluster_means).outer(twin_theory)).set_symmetry(D6, D6).angle.data"
```
Comment (pc494) [inline 03 - Clustering Misorientations.ipynb]: I am fairly sure that this isn't guaranteed to work.

File: 03 - Clustering Misorientations.ipynb [hunk 4]
```diff
@@ -165,120 +246,296 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mismisori = (~twin_theory).outer(cluster_means)\n",
+    "mismisori_equiv = D6.outer(~twin_theory).outer(D6).outer(D6).outer(cluster_means).outer(D6)\n",
```
Comment (pc494) [inline 03 - Clustering Misorientations.ipynb:426]: I'm pretty sure this is the correct idea.

**Review by pc494 — COMMENTED** [review]
Some notes.

File: 03 - Clustering Misorientations.ipynb (hunk 3 above)
Comment (dnjohnstone) [reply 03 - Clustering Misorientations.ipynb]: Yeah I agree with that, hence the second go with symmetry

File: 03 - Clustering Misorientations.ipynb (hunk 4 above)
Comment (dnjohnstone) [reply 03 - Clustering Misorientations.ipynb:426]: Yeah, but this is the one that still seems to have an issue with order...

File: 03 - Clustering Misorientations.ipynb (hunk 1 above)
Comment (dnjohnstone) [reply 03 - Clustering Misorientations.ipynb]: I think this is part of the minefield of different conventions for hexagonal systems and also that symmetry relationships with 3-index notation are not so obvious as with 4-index for hexagonal systems...

**Pull request merged**